tensorboard/components/experimental/plugin_lib/plugin-guest.ts

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+namespace tb_plugin.lib.internal {
```

Comment:
thinking out loud: now that these symbols will be exposed on window, I wonder if we should use more dangerous looking namespace to tell people to not use this directly. `tb_plugin.lib.DO_NOT_USE_internal`? :D

Reply:
Done.